lib/lotus/commands/runner.rb

```diff
@@ -0,0 +1,48 @@
+require 'lotus/environment'
+require 'lotus/application'
+
+module Lotus
+  module Commands
+    # Evaluate a file or expression with the preloaded environment and application.
+    #
+    # It is run with:
+    #
+    #   `bundle exec lotus runner ./my_script.rb`
+    #
+    # or
+    #
+    #   `bundle exec lotus runner 'puts Lotus.env'`
+    #
+    # @since x.x.x
+    # @api private
+    class Runner
+      # @param options [Hash] Environment's options
+      # @param expression_or_file [String] Expression or file to evaluate
+      #
+      # @since x.x.x
+      # @see Lotus::Environment#initialize
+      # @see Lotus::Environment#require_application_environment
+      # @see Lotus::Application.preload!
+      def initialize(options, expression_or_file)
+        @expression_or_file = expression_or_file
+        raise ArgumentError.new("Provide file or expression to execute") if @expression_or_file.nil?
```

Comment:
@davydovanton It will throw a `NoMethorError`, which isn't user friendly, and it will perceived as a bug in Lotus. It's better a message that explains what's the problem, like we do in this case.